### PR TITLE
Added GUI

### PR DESCRIPTION
**Pull request by JJSnader**
It's not fully functional, but there are other tasks for fleshing it out. The core of the form is there, enough that the requirements of the Initial GUI Setup task are satisfied.

### CODE REVIEW

File: app/src/main/java/xterminators/spellingbee/GuiController.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import java.awt.*;
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.nio.file.Paths;
+
+public class GuiController {
+
+    private JFrame m_frame;
+    private JPanel m_panel;
+    private JTextField tbGuess;
```
Comment (JonathanMathBoi) [inline app/src/main/java/xterminators/spellingbee/GuiController.java:13]: It's kind of difficult to tell what this variable is at first glance, is it possible to give it a more descriptive (even if slightly longer) name?

File: app/src/main/java/xterminators/spellingbee/GuiController.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    private UserFunctions m_functions;
```
Comment (JonathanMathBoi) [inline app/src/main/java/xterminators/spellingbee/GuiController.java:14]: Is this the same UserFunctions we used the the sprint 1 CLI? If so, keep in mind that it prints to the console as output, a GUIView class might be better (if needed at all).

**Review by JonathanMathBoi — APPROVED** [review]
Made a few comments about member variables, but overall everything looks good.

File: app/src/main/java/xterminators/spellingbee/GuiController.java (hunk 2 above)
Comment (JJSnader) [reply app/src/main/java/xterminators/spellingbee/GuiController.java:14]: This is the same UserFunctions. I thought maybe console output wouldn't matter if the user is using the GUI, but a separate class would avoid any output issues.

**Pull request merged**